### ISSUE TITLE
A rule with the dead domain is skipped

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/DeadDomainsLinter#11]
```adblock
@@||filmisub.com/templates/filmisub/adblock/adframe.ad-target.js
```

<details>
<summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/DeadDomainsLinter/assets/8361299/c645a858-c5db-4933-95e4-e4311c7af64e)

</details>

<details>
<summary>Rules</summary>

```adblock
! https://github.com/AdguardTeam/AdguardFilters/issues/102223
@@||filmisub.com/templates/filmisub/adblock/adframe.ad-target.js
filmisub.com#%#//scriptlet("set-constant", "noAdBlock", "true")
filmisub.com#%#//scriptlet("set-constant", "adsNotBlocked", "true")
filmisub.com#%#//scriptlet("set-constant", "detis", "noopFunc")
filmisub.com#%#//scriptlet('prevent-fetch', 'adskeeper')
!+ PLATFORM(ios, ext_android_cb)
@@||jsc.adskeeper.com/*/filmisub.com.*.js$xmlhttprequest,domain=filmisub.com
```

</details>

**Comment by ameshkov** [comment]
Ah, it does not catch `||domain.com/` now, I'll fix it.

**Comment by ameshkov** [comment]
Actually, I missed much more than just that.

`@@||` rules were also ignored.
